tests/unittest_protocols.py

```diff
@@ -297,7 +297,7 @@ def test_assigned_stmts_match_mapping():
                 pass
         """
         )
-        match_mapping: nodes.MatchMapping = assign_stmts.pattern  # type: ignore
+        match_mapping: nodes.MatchMapping = assign_stmts.pattern
```

Comment:
The `type: ignores` here are only unused since the method isn't annotated and thus not type checked.